packages/util/src/subscribe.ts

```diff
@@ -78,7 +78,7 @@ class ObserverProxy<T> implements Observer<T> {
   // Micro-task scheduling by calling task.then().
   private task = Promise.resolve();
   private finalized = false;
-  private finalError: Error;
+  private finalError?: Error;
```

Comment:
What does "?" mean here? It could be undefined?

Reply:
Yeah, either that or it should be `Error | null` and initialized to null, because it's not initialized and is only assigned if there is an error.  There is a falsy check later when it is used.  What do you think?